README.md

```diff
@@ -1,7 +1,28 @@
 
-# ukim-auth-checker-frontend
+# UKIM Auth Checker Frontend
 
-UKIM AUTH CHECKER FRONTEND
+This repository contains the necessary code to display user journey for UKIM Auth Checker Frontend
+
+The codebase uses the none scaffolding route with no database
+
+### Overview
+
+The Frontend is for a new UK Internal Market Scheme (UKIMS) for internal trade that enables traders to self-assess their goods’ destination as At Risk (AR) or Not at Risk (NAR).
+It will take in one UKIM number, make a call to the backend (https://github.com/hmrc/pds-auth-checker-api), returning the UKIM numbers validity.
```

Comment:
Sorry, missed this as well. It will take in an EORI number, and returns whether the EORI number is associated with a valid or invalid UKIMs Authorisation